src/App.tsx

```diff
@@ -1,17 +1,24 @@
 import { createRoot } from 'react-dom/client';
 import { reatomContext } from '@reatom/react';
 import { StrictMode } from 'react';
+import { appInit } from '~core/app/init';
 import { store } from '~core/store/store';
-import { AuthWrapper } from '~core/auth';
 import { Router } from '~core/router';
 
 const root = createRoot(document.getElementById('root')!);
-root.render(
-  <StrictMode>
-    <reatomContext.Provider value={store}>
-      <AuthWrapper>
+
+appInit().then(() => {
+  root.render(
+    import.meta.env?.VITE_DEBUG_DISABLE_REACTSTRICTMODE ? (
```

Comment:
is my guess right that disabling react strict mode will disable double-invoking effects and renders?

Reply:
yes, and it's for dev only. need option for disabling to have prod-like behavior